kernel/enter.c

```diff
@@ -564,6 +564,27 @@ static int _mount_rootfs(myst_kernel_args_t* args, myst_fstype_t fstype)
     return ret;
 }
 
+#if defined(MYST_ENABLE_HOSTFS)
+#define ETC_RESOLV_CONF "/etc/resolv.conf"
+static int _mount_host_resolv_conf()
```

Comment:
Auto mount `/etc/resolv.conf` without opt-in effectively overrides the hardcoded value inserted by `AppBuilder` (also no opt-in). Is that the design intention?

Reply:
yes

Reply:
Changed.